style.css

```diff
@@ -681,6 +685,126 @@ footer img {
   width: 327px;
 }
 
+/* Pop up */
+
+.blur {
+  filter: blur(100px);
+}
+
+.pop-up-section {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #f3f1f1;
+  display: none;
+  justify-content: center;
+  z-index: 20;
+  backdrop-filter: blur(2px);
+}
```

Comment:
Please be sure that your design comes as close as possible to Figma's. Try giving your popup elements such as your x button a bit more margin. Also, please make your color for the popup the same as the design. Another important element to adjust is the color of the orange buttons. Note that on Figma's the color for the text is white. Kindly make the mentioned changes so that your project passes all of the requirements ✔

Figma's
![Captura](https://user-images.githubusercontent.com/15807343/162072744-24055f25-6975-4913-8bba-621b68ab3539.PNG)


Yours
![image](https://user-images.githubusercontent.com/15807343/162072229-827503ad-402e-4004-b6d3-97fb4130c7b5.png)

Reply:
Thank you, I got this fixed!